src/webots/gui/WbMainWindow.cpp

```diff
@@ -1639,11 +1639,17 @@ void WbMainWindow::upload(char type) {
 
   QNetworkReply *reply = WbNetwork::instance()->networkAccessManager()->post(request, multiPart);
 
-  mUploadProgressDialog = new QProgressDialog(tr("Uploading on %1...").arg(uploadUrl), "Cancel", 0, 100, this);
+  QString cancelText = "cancel";
```

Comment:
The cancel buttons are usually written with an uppercase letter, more over this can be const:
```suggestion
  const QString cancelText = "Cancel";
```